lib/ABI/Analyses/ConvertToCABIFunctionType.cpp

```diff
@@ -189,16 +189,36 @@ class ConvertToCABIFunctionType {
       // TODO: extend this list.
       static constexpr std::array ABIsTheConversionIsEnabledFor = {
         model::ABI::SystemV_x86_64,
+        model::ABI::Microsoft_x86_64,
+        model::ABI::Microsoft_x86_64_vectorcall,
         model::ABI::SystemV_x86,
         model::ABI::SystemV_x86_regparm_3,
         model::ABI::SystemV_x86_regparm_2,
         model::ABI::SystemV_x86_regparm_1,
         model::ABI::Microsoft_x86_cdecl,
+        model::ABI::Microsoft_x86_cdecl_gcc,
         model::ABI::Microsoft_x86_fastcall,
+        model::ABI::Microsoft_x86_fastcall_gcc,
         model::ABI::Microsoft_x86_stdcall,
+        model::ABI::Microsoft_x86_stdcall_gcc,
         model::ABI::Microsoft_x86_thiscall,
         model::ABI::Microsoft_x86_vectorcall,
         model::ABI::AAPCS
+
+        // There are known issues
+        // model::ABI::AAPCS64,
+
+        // There are known issues
+        // model::ABI::SystemV_MIPS_o32,
+        // model::ABI::SystemV_MIPSEL_o32
+
+        // Unable to reliably test: no easy access to a compiler
+        // model::ABI::Microsoft_x86_64_clrcall,
+        // model::ABI::Microsoft_x86_clrcall,
```

Comment:
Please let's purge untested things, I don't think we want to support these for now.

Reply:
I just listed them here to keep track of what we are testing and what we don't. I think there's a value in explicitly specifying "do not rely to much on this if you use X ABI" in the body of the analysis.

Reply:
I'd prefer to drop `clrcall` ABIs entirely, having them there without ever testing them is mostly noise.

Reply:
Ah, you were talking about `clrcall`, not about the comments in itself. My bad then.

No complaints on dropping `clrcall`.

Reply:
Dropped. See the new `ABI: drop incomplete `clrcall` support` commit.